src/main/java/com/capgemini/brahma/Application.java

```diff
@@ -1,13 +1,15 @@
 package com.capgemini.brahma;
 
-import com.capgemini.camel.metrics.publisher.configuration.MetricsServletConfiguration;
+import org.apache.camel.spring.boot.FatJarRouter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.apache.camel.spring.boot.FatJarRouter;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Import;
 
-@Import({MetricsServletConfiguration.class})
+import com.capgemini.brahma.config.JettyServerCustomizer;
```

Comment:
can you move these back up? we only separate out project-local code down to this block